packages/budget/src/components/CardCollection/index.js

```diff
@@ -19,11 +20,11 @@ const CardCollection = () => (
           decision-making!
         </p>
       </div>
-      <div className="ten-year__wrapper">
```

Comment:
There are some styles in CardCollection.css for `.ten-year__wrapper`, are they needed? If not, want to remove them?

Reply:
yep!